traitlets/traitlets.py

```diff
@@ -319,6 +320,23 @@ class that does the following:
        accept superclasses for :class:`This` values.
     """
 
+    def instance_init(self, obj):
```

Comment:
Done (and for this_class too)
- although what is set by the metaclass is an instance attribute. Let me know if you want me to remove it.